tools/packaging/release/release.sh

```diff
@@ -114,9 +114,23 @@ function _create_new_release()
 
 	_create_our_own_notes
 
+	# This automatically creates the ${RELEASE_VERSION} tag in the repo
 	gh release create ${RELEASE_VERSION} \
 		--title "Kata Containers ${RELEASE_VERSION}" \
-		--notes-file "/tmp/our_notes_${RELEASE_VERSION}"
+		--notes-file "/tmp/our_notes_${RELEASE_VERSION}" \
```

Comment:
I guess this will clash with https://github.com/kata-containers/kata-containers/pull/9321